<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>passWindow</class>
 <widget class="QMainWindow" name="passWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="announce">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Meltix Regular Demo</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Change Password</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="Gnum">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>10</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Meltix Regular Demo</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:transparent;
     border-radius: 20px;
     border: 1px solid ;
     width: 20px;
     height: 1px;
     padding-left: 10px;
border-top-color: transparent;
border-right-color: transparent;
border-left-color: transparent;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Meltix Regular Demo</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <item>
            <property name="text">
             <string>  Password Type</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Default OUP</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Card Password</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>198</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="Gpre">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Meltix Regular Demo</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:transparent;
     border-radius: 20px;
     border: 1px solid ;
     width: 20px;
     height: 1px;
     padding-left: 10px;
border-top-color: transparent;
border-right-color: transparent;
border-left-color: transparent;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BvisiablePre">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">
background-color: transparent;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="Gnew">
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Meltix Regular Demo</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:transparent;
     border-radius: 20px;
     border: 1px solid ;
     width: 20px;
     height: 1px;
     padding-left: 10px;
border-top-color: transparent;
border-right-color: transparent;
border-left-color: transparent;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BvisiableNew">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">
background-color: transparent;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Grenrew">
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Meltix Regular Demo</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:transparent;
     border-radius: 20px;
     border: 1px solid ;
     width: 20px;
     height: 1px;
     padding-left: 10px;
border-top-color: transparent;
border-right-color: transparent;
border-left-color: transparent;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BvisiableCon">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">
background-color: transparent;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="rightMargin">
          <number>30</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QPushButton" name="Bverify">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QComboBox" name="CBgetCard">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Meltix Regular Demo</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <item>
            <property name="text">
             <string>Choose Card</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Bback">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Meltix Regular Demo</family>
             <pointsize>12</pointsize>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-top: 50px;
color: rgb(85, 255, 255);
background-color: rgb(0, 0, 0);
  text-align: center;
  border-radius:30px;
box-sizing: border-box;
  margin: 0;
border-color: rgb(0, 255, 255);
  padding: 0;</string>
           </property>
           <property name="text">
            <string> Back</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>180</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="BChange">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>2</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Meltix Regular Demo</family>
             <pointsize>12</pointsize>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-top: 50px;
color: rgb(85, 255, 255);
background-color: rgb(0, 0, 0);
  text-align: center;
  border-radius:30px;
box-sizing: border-box;
  margin: 0;
border-color: rgb(0, 255, 255);
  padding: 0;</string>
           </property>
           <property name="text">
            <string> Change</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="error">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Meltix Regular Demo</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;
color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
